js/ui/data_grid/ui.data_grid.adaptivity.js

```diff
@@ -788,6 +789,17 @@ gridCore.registerModule("adaptivity", {
                     this.callBase(rowIndex);
                 },
 
+                deleteRow: function(rowIndex) {
+                    var expandedKey = this._dataController._adaptiveExpandedKey,
+                        expandedRowIndex = gridCoreUtils.getIndexByKey(expandedKey, this._dataController.items());
```

Comment:
I think that using getKeyByRowIndex and isAdaptiveDetailRowExpanded is better way then using private fields and utility method